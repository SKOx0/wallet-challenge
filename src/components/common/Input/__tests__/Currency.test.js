import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import IntlCurrencyInput from 'react-intl-currency-input';
import CurrencyInput from '../Currency';

describe('<CurrencyInput />', () => {
  it('deve ser igual ao snapshot, com configurações padrões', () => {
    const renderedComponent = shallow(<CurrencyInput />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });

  it('deve passar informações do input de currency', () => {
    const renderedComponent = shallow(<CurrencyInput />);

    expect(renderedComponent.find(IntlCurrencyInput)).toBeDefined();
  });
});
