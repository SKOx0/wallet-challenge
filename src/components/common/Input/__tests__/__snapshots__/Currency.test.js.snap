// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyInput /> deve ser igual ao snapshot, com configurações padrões 1`] = `
.emotion-0 {
  height: 50px;
  padding: 15px;
  width: 100%;
  border-radius: 3px;
  border-width: 1px;
  border-color: #D2D2D2;
  font-size: 1.0rem;
}

.emotion-0:focus {
  outline: none;
}

.emotion-0[readonly],
.emotion-0 [disabled] {
  opacity: 0.5;
}

<CurrencyComponent
  className="emotion-0"
/>
`;
